### PR TITLE
Zeroing out gradients recipe

### PR DESCRIPTION
**Pull request by CamiWilliams**
Intro-level recipe for zeroing out gradients in PyTorch. Showcases CIFAR10 to train on.

### CODE REVIEW

**Comment by CamiWilliams** [comment]
Updated py according to script.
